Description: ffmpeg doesn't support pkg-config's Requires.private and
Libs.private, so we merge those into the public dependencies instead.

--- a/sharpyuv/libsharpyuv.pc.in
+++ b/sharpyuv/libsharpyuv.pc.in
@@ -7,5 +7,4 @@
 Description: Library for sharp RGB to YUV conversion
 Version: @PACKAGE_VERSION@
 Cflags: -I${includedir}
-Libs: -L${libdir} -l@webp_libname_prefix@sharpyuv
-Libs.private: -lm @PTHREAD_CFLAGS@ @PTHREAD_LIBS@
+Libs: -L${libdir} -l@webp_libname_prefix@sharpyuv -lm @PTHREAD_CFLAGS@ @PTHREAD_LIBS@
--- a/src/demux/libwebpdemux.pc.in
+++ b/src/demux/libwebpdemux.pc.in
@@ -6,6 +6,6 @@
 Name: libwebpdemux
 Description: Library for parsing the WebP graphics format container
 Version: @PACKAGE_VERSION@
-Requires.private: libwebp >= 0.2.0
+Requires: libwebp >= 0.2.0
 Cflags: -I${includedir}
 Libs: -L${libdir} -l@webp_libname_prefix@webpdemux
--- a/src/libwebpdecoder.pc.in
+++ b/src/libwebpdecoder.pc.in
@@ -7,5 +7,4 @@
 Description: Library for the WebP graphics format (decode only)
 Version: @PACKAGE_VERSION@
 Cflags: -I${includedir}
-Libs: -L${libdir} -l@webp_libname_prefix@webpdecoder
-Libs.private: -lm @PTHREAD_CFLAGS@ @PTHREAD_LIBS@
+Libs: -L${libdir} -l@webp_libname_prefix@webpdecoder -lm @PTHREAD_CFLAGS@ @PTHREAD_LIBS@
--- a/src/libwebp.pc.in
+++ b/src/libwebp.pc.in
@@ -6,7 +6,6 @@
 Name: libwebp
 Description: Library for the WebP graphics format
 Version: @PACKAGE_VERSION@
-Requires.private: libsharpyuv
+Requires: libsharpyuv
 Cflags: -I${includedir}
-Libs: -L${libdir} -l@webp_libname_prefix@webp
-Libs.private: -lm @PTHREAD_CFLAGS@ @PTHREAD_LIBS@
+Libs: -L${libdir} -l@webp_libname_prefix@webp -lm @PTHREAD_CFLAGS@ @PTHREAD_LIBS@
--- a/src/mux/libwebpmux.pc.in
+++ b/src/mux/libwebpmux.pc.in
@@ -6,7 +6,6 @@
 Name: libwebpmux
 Description: Library for manipulating the WebP graphics format container
 Version: @PACKAGE_VERSION@
-Requires.private: libwebp >= 0.2.0
+Requires: libwebp >= 0.2.0
 Cflags: -I${includedir}
-Libs: -L${libdir} -l@webp_libname_prefix@webpmux
-Libs.private: -lm
+Libs: -L${libdir} -l@webp_libname_prefix@webpmux -lm
